src/Entities/IssueDetails.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace ITBugTracking\Entities;
+
+use ITBugTracking\Entities\Issue;
+
+class IssueDetails extends Issue
+{
+    public array|null $comments;
+    public string $reporter;
+    public int $department;
+    public string $description;
```

Comment:
Look at the database, this is a `string | null` field.

Reply:
done